### PR TITLE
add metrics and logs for s3

### LINKED ISSUE #230: Feature: More transparency to s3 plugin
S3 call has no timeout and to metrics which counts committed batches. 
Add some.

### PR DESCRIPTION
**Pull request by ansakharov**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #230

### CODE REVIEW

File: plugin/output/s3/s3.go [hunk 1]
```diff
@@ -491,8 +499,10 @@ func (p *Plugin) uploadWork() {
 	for compressed := range p.uploadCh {
 		sleepTime := attemptInterval
 		for {
+			p.logger.Infof("starting upload s3 object: %s to bucket: %s, time: %s", compressed.fileName, compressed.bucketName, time.Now().Format(time.RFC3339))
```
Comment (vano144) [inline plugin/output/s3/s3.go:502]: You can change log format and time.now will be useless

**Review by vano144 — APPROVED** [review]

**Review by vadimalekseev — APPROVED** [review]

File: plugin/output/s3/s3.go [hunk 2]
```diff
@@ -215,6 +218,10 @@ type Config struct {
 	// > DynamicBucketsLimit regulates how many buckets can be created dynamically.
 	// > Prevents problems when some random strings in BucketEventField where
 	DynamicBucketsLimit int `json:"dynamic_buckets_limit" default:"32"` // *
+	// > @3@4@5@6
+	// > UploadTimeout sets max upload timeout to s3.
+	UploadTimeout  cfg.Duration `json:"upload_timeout" default:"1m" parse:"duration"`
```
Comment (vitkovskii) [inline plugin/output/s3/s3.go:223]: Comment should looks like:
```
	// > @3@4@5@6
	// > 
	// > Sets upload timeout.
```

Empty line is missing. Don't repeat parameter name. Also please format other parameters as well.

**Review by vitkovskii — CHANGES_REQUESTED** [review]

File: plugin/output/s3/s3.go (hunk 2 above)
Comment (ansakharov) [reply plugin/output/s3/s3.go]: Fixed.

File: plugin/output/s3/s3.go (hunk 1 above)
Comment (ansakharov) [reply plugin/output/s3/s3.go]: Added to all logs.

File: plugin/output/file/file.go [hunk 3]
```diff
@@ -259,7 +259,7 @@ func (p *Plugin) sealUp() {
 	if err := oldFile.Close(); err != nil {
 		p.logger.Panicf("could not close file: %s, error: %s", oldFile.Name(), err.Error())
 	}
-	logger.Errorf("sealing in %d, newFile: %s", time.Now().Unix(), newFileName)
+	logger.Infof("sealing in %d, newFile: %s", time.Now().Unix(), newFileName)
```
[the remark this answers is not in the document]
Comment (ansakharov) [reply plugin/output/file/file.go]: changed logging messages.

**Review by vitkovskii — APPROVED** [review]

**Pull request merged**